app/assets/javascripts/application.js

```diff
@@ -5,4 +5,4 @@
 //= require_tree .
 
 window.GOVUK.stickAtTopWhenScrolling.init();
-window.GOVUK.stopScrollingAtFooter.init();
+window.GOVUK.stopScrollingAtFooter.addEl($('.page-contents'));
```

Comment:
I wrote the below comment, then looked at the PR on govuk_frontend_toolkit, and my comment isn't related to what you're changing. So.. ignore this.. but maybe for one day in the future.

I like the passing of the element into `addEl()`. How about doing something similar with the `init()` above?

eg.

```
window.GOVUK.stickAtTopWhenScrolling.init($(".js-stick-at-top-when-scrolling"));
```

Or.. Could the `data-module` technique be used?